prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml

```diff
@@ -1,7 +1,7 @@
 
```

Comment:
The same should be done for release-1.1 branch.

Reply:
https://github.com/istio/test-infra/pull/1170